data/gamemoded.8

```diff
@@ -110,6 +110,12 @@ softrealtime=auto
 ; the value will be negated and applied as a nice value
 renice = 4
 
+; By default, GameMode adjusts the iopriority of clients to BE/0, you can put any value
+; between 0 and 7 here (with 0 being highest priority), or one of the special values
+; "off" (to disable) or "reset" (to restore Linxu default bahavior based on CPU priority),
```

Comment:
Linux

Reply:
behaviour

Reply:
> behaviour

or behavior

Reply:
I'm used to american English better, so I use "behavior". But I'm fixing this...

Reply:
Yes, i agree, but you spelled "behavior" as "bahavior". ;)

Reply:
What!? Oh noes! Will fix... :-)